### PR TITLE
[READ THIS PR] Changeling balance changes 

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request

Rebalances changelings to now be closer to on par with traitors (Though, i believe they are still more powerful). Read the changelog for full changes. Making resonant screech respect hearing protection is one of these changes.
Most significantly, this changes how changelings will typically be reviving- changelings now cannot revive when husked, but have the Last Resort ability by default, opening up new avenues to kill or incapacitate lings
Second most importantly , this removes changeling teams, making the antag a bit less of a drag to play, and encouraging competitive gameplay. Hivemind is no longer a default changeling ability to reflect this
most of the other changes will be in the changelog below, but here are the significant ones:

**fleshmend** is now a toggleable passive heal instead of an active ability. While active, it will heal brute damage at decent speed, and slowly heal cell and burn damage. it will not work while the user is on fire, and slows chem production. it's not much useful for combat, but it'll take care of the wounds afterwords without slowing you down

**anatomic panacea** now no longer synthesizes a bunch of minor healing chems in the host. Instead, it violently ejects all reagents within the ling around them. All of the effects other than radiation removal remain. It won't cure incredibly resistant diseases though, nor will it cure diseases that aren't quite diseases (transformation diseases, mostly)

**Spread Infestation** has been totally gutted and changed. It now spawns a teratoma monkey, which can be controlled by a player, with the directive to just cause chaos. the chemical and point costs have been increased to reflect this

**last resort** is now a default ability. it no longer causes explosions on use, but confuses those around the changeling a bit more. It is also useable whilst unconscious, and headslugs can now impregnate unconscious bodies, on top of already dead people.

**Shield** was a loud item that encouraged murderboning both in its functionality and mechanics (requiring absorbs for free hits, etc), and it has been removed

## Why It's Good For The Game

changelings are donut steel anime level of power, this tones them down just a bit. also, they now actually have to fear burn damage. also, teamlings are borked as fuck

may make a new ability pr based on feedback if i feel like it

## Changelog
:cl:
del: Removed team lings
del: removed ling's shield
del: Since it was near useless, the hivemind link ability was axed
tweak: changelings no longer have a cap on genomes
tweak: Hivemind and Regenerate are no longer default changeling abilities
tweak: headslugs can now be used whilst incapacitated or dead, and no longer cause an explosion. 
tweak: spread infestation now spawns a single, reskinned, player controlled tumor monkey to cause mayhem
tweak: pheremone receptors no longer slows chem regen
tweak: mimic voice no longer slows chem regen
tweak: augmented eyesight no longer gives flash protection while inactive
tweak: changelings cannot absorb targets that cannot die
tweak: absorbing other lings no longer gives you all of their points, instead giving you a flat +2, and not increasing your chem storage.
balance: chameleon skin cost halved, and it no longer costs chems to toggle
balance: fleshmend is now a passive heal that slows regen, instead of a quick, active combat heal
balance: anatomic panacea has been buffed and now no longer relies on spawning healing chems in the user
balance: Changelings can no longer revive when husked
balance: armblade damage reduced by 5. it now deals 20 damage, slightly more than a dark blessing
balance: resonant screech respects hearing protection (FINALLY)
imageadd: headslugs are now headspiders, and no longer look like weird purple things
/:cl:

### CODE REVIEW

**Comment by Isy232** [comment]
Why nerf lings absorbing other lings..., its already hard as shit to pull off and if anything makes the round more calm as it removes antagonists. Its a high risk high reward gamble to eat another ling.

Why nerf augmented eye-sight, its 2 points for flash immunity, you can just wear a pair of sunglasses. This ability is already fairly bad, this is just nerfing it into the ground. 

I do agree with all of the other changes, however please reconsider nerfing augmented eye-sight especially.

**Comment by zeskorion** [comment]
> Why nerf lings absorbing other lings..., its already hard as shit to pull off and if anything makes the round more calm as it removes antagonists. Its a high risk high reward gamble to eat another ling.

because lings themselves are far easier to catch now, and ling-eating essentially doubles the danger of a single changeling. imagine if only traitors could use traitor uplinks, and they were always unlocked!

> Why nerf augmented eye-sight, its 2 points for flash immunity, you can just wear a pair of sunglasses. This ability is already fairly bad, this is just nerfing it into the ground.

x-ray vision is the best vision option in the game, and as far as points go, this is far better than traitor's 3 point thermal vision goggles

**Comment by Isy232** [comment]
> because lings themselves are far easier to catch now, and ling-eating essentially doubles the danger of a single changeling. imagine if only traitors could use traitor uplinks, and they were always unlocked!
For each ling absorbed you gain half of that changelings power, while it does increase your own power it still decreases the overall power of the antagonists by half a changeling. Also because changelings get stronger from eating each other its a good incentive for changelings to fight each other.

If you eat 4 other changelings and get a chemical capacity of 200 something it makes you strong as hell but even then its still possible to kill you. Perhaps not with standard security officer gear, but sepia slimes, syringe guns, tech shells, mechs, grenades. All of these make you quite close to being even with a powerful changeling and by the time a changeling eats so many others it would likely be that far into the round. With changeling revive being changed you only need to keep a changeling down for long enough to flamethrower or throw them into a hellfoam grenade to permanently kill them.

If you make absorbing a changeling barely have any reward there won't be any point in doing so, so changelings are much more likely to freely collaborate with each other, maybe even leading to them being even stronger.

>x-ray vision is the best vision option in the game, and as far as points go, this is far better than traitor's 3 point thermal vision goggles

Yeah, x-ray vision is good but you don't have flash protection while x-ray is on. I still do not see why this nerf is needed at all.

**Comment by VictorVow** [comment]
You even get a reduction in eye protection such that you can get flashed through sunglasses with x-ray vision on.

**Comment by zeskorion** [comment]
> You even get a reduction in eye protection such that you can get flashed through sunglasses with x-ray vision on.

you can turn it off



> > because lings themselves are far easier to catch now, and ling-eating essentially doubles the danger of a single changeling. imagine if only traitors could use traitor uplinks, and they were always unlocked!
> > For each ling absorbed you gain half of that changelings power, while it does increase your own power it still decreases the overall power of the antagonists by half a changeling. Also because changelings get stronger from eating each other its a good incentive for changelings to fight each other.
> 
> If you eat 4 other changelings and get a chemical capacity of 200 something it makes you strong as hell but even then its still possible to kill you. Perhaps not with standard security officer gear, but sepia slimes, syringe guns, tech shells, mechs, grenades. All of these make you quite close to being even with a powerful changeling and by the time a changeling eats so many others it would likely be that far into the round. With changeling revive being changed you only need to keep a changeling down for long enough to flamethrower or throw them into a hellfoam grenade to permanently kill them.
> 
> If you make absorbing a changeling barely have any reward there won't be any point in doing so, so changelings are much more likely to freely collaborate with each other, maybe even leading to them being even stronger.

sepia slimes can be avoided, in fact, pretty much the only one thing on that list that cant be beaten without significant robustness is mechs. remember, lings can still use stationside gear

> > x-ray vision is the best vision option in the game, and as far as points go, this is far better than traitor's 3 point thermal vision goggles
> 
> Yeah, x-ray vision is good but you don't have flash protection while x-ray is on. I still do not see why this nerf is needed at all.

it's the same with traitor xray, though

**Comment by zeskorion** [comment]
dammit travis, you idiot, failing your isntall

**Comment by ike709** [comment]
Vote passed but I haven't had time to review this.

**Comment by jlsnow301** [comment]
> del: Removed team lings

Thank GOD. This has been bugged for centuries, if it isn't impossible enough to pull off already

**Comment by Isy232** [comment]
> sepia slimes can be avoided, in fact, pretty much the only one thing on that list that cant be beaten without significant robustness is mechs. remember, lings can still use stationside gear

Yes, but while they cant use stationside gear while using their changeling abilities. You cant armblade someone while in a mech. All of these other things do require significant robustness or luck to pull off, but all have the potential to beat a changeling no matter how strong they are. If you kill 4 other changelings and get a chemical capacitry of 200, your still far less strong than 2 or 3 robust changelings working together.

Also, while your busy setting up your objectives you cannot be working on items that require time to get, however in some cases you can just steal the fruits of another's work.

>it's the same with traitor xray, though

For a traitor their goggles cost 3/20 TC, 15% of their Points.
For a changeling, this ability costs 2/10, 20% of their Points.
5% of your special gear for flash protection when the main ability is disabled seems pretty fair.

You have nerfed a lot of things changelings could use to murderbone and buffed things that allow them to work more stealthily, but I think that this ability is important to even those playing changeling stealthily as if you have to assassinate some head of staff or even just someone who decided to pick up a flash you could pretty much be screwed instantly.

**Comment by zeskorion** [comment]
> > sepia slimes can be avoided, in fact, pretty much the only one thing on that list that cant be beaten without significant robustness is mechs. remember, lings can still use stationside gear
> 
> Yes, but while they cant use stationside gear while using their changeling abilities. You cant armblade someone while in a mech. All of these other things do require significant robustness or luck to pull off, but all have the potential to beat a changeling no matter how strong they are. If you kill 4 other changelings and get a chemical capacitry of 200, your still far less strong than 2 or 3 robust changelings working together.
> 
> Also, while your busy setting up your objectives you cannot be working on items that require time to get, however in some cases you can just steal the fruits of another's work.
> 
> > it's the same with traitor xray, though
> 
> For a traitor their goggles cost 3/20 TC, 15% of their Points.
> For a changeling, this ability costs 2/10, 20% of their Points.
> 5% of your special gear for flash protection when the main ability is disabled seems pretty fair.
> 
> You have nerfed a lot of things changelings could use to murderbone and buffed things that allow them to work more stealthily, but I think that this ability is important to even those playing changeling stealthily as if you have to assassinate some head of staff or even just someone who decided to pick up a flash you could pretty much be screwed instantly.

actually, some abilities do work inside mechs, iirc . you're looking at changelings as if they should be extremely powerful, but having counters is good- look a traitors, they're just normies with some gamer gear

**Comment by Isy232** [comment]
> actually, some abilities do work inside mechs, iirc . you're looking at changelings as if they should be extremely powerful, but having counters is good- look a traitors, they're just normies with some gamer gear

I think the screech and EMP does, but if you EMP, you damage your mech and if you screech, the same could be done even better by a clusterbang.

Changelings are strong but also hard to play. Its difficult to get all of the stealth going correctly and most of their abilities are only effective in a 1 on 1 fight.

File: code/datums/status_effects/buffs.dm [hunk 1]
```diff
@@ -421,11 +421,9 @@
 
 
 //Used by changelings to rapidly heal
-//Heals 10 brute and oxygen damage every second, and 5 fire
 //Being on fire will suppress this healing
 /datum/status_effect/fleshmend
 	id = "fleshmend"
-	duration = 100
```
Comment (qwertyquerty) [inline code/datums/status_effects/buffs.dm:428]: You are aware this makes the effect never end until turned off by the user right?

Comment (zeskorion) [reply code/datums/status_effects/buffs.dm:428]: yes, that was intentional. it's a passive heal now, much slower

**Pull request merged**